### PR TITLE
Introducing postgres to faucet

### PR DESCRIPTION
**Pull request by mutantcornholio**
Using TypeORM as probably most popular and mature TS ORM out there

* Removed DripperStorage and its tests, as all logic inside of it became
super trivial.
* Renamed `isValid`/`saveData` functions from DripperStorage to more
explicit `saveDrip`/`hasDrippedToday`.
* Changed DripRequestHandler tests to more unit-tests approach with
mocking dripperStorage.
* Upgrading prettier (that was required to use with decorators properly)
results in insignificant changes accross some files (app.css, app.html,
etc.)

A part of https://github.com/paritytech/polkadot-testnet-faucet/issues/335

### CODE REVIEW

**Comment by mutantcornholio** [comment]
> What is your opinion regarding [Prisma](https://www.prisma.io/)

Another option worth considering

File: package.json [hunk 1]
```diff
@@ -62,20 +62,20 @@
     "bs58": "^5.0.0",
     "confmgr": "^1.0.8",
     "cors": "^2.8.5",
-    "crypto": "^1.0.1",
```
Comment (rzadp) [inline package.json:65]: Why is this removed? It's used directly in `src/dripper/dripperStorage.ts`

Comment (rzadp) [reply package.json:65]: Ah ok, I didn't realize this was a `node` thing.

**Review by rzadp — APPROVED** [review]
